Administrative Assistant, a new RP / Support centric "command" role

noctyrnal opened the pull request:
<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- The text between the arrows are comments - they will not be visible on your PR. -->

## About the PR
<!-- What did you change in this PR? -->
Adds the Administrative Assistant role to all stations, acting as both a secretary for command and as a worker on the bridge, answering faxes , grabbing coffees for command members and so on. Intended to support command members in more menial tasks, and to provide unique RP circumstances.

## Why / Balance
<!-- Why was it changed? Link any discussions or issues here. Please discuss how this would affect game balance. -->
Makes command's life a bit easier to have a person dedicated to helping them do the small things

## Technical details
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->
None

## Media
<!-- 
PRs which make ingame changes (adding clothing, items, new features, etc) are required to have media attached that showcase the changes.
Small fixes/refactors are exempt.
Any media may be used in SS14 progress reports, with clear credit given.

If you're unsure whether your PR will require media, ask a maintainer.

Check the box below to confirm that you have in fact seen this (put an X in the brackets, like [X]):
-->

![f77bc14897d700be8c36fde6fb72c6af](https://github.com/DeltaV-Station/Delta-v/assets/99278536/d7f8a859-43ae-43ab-8f53-6839d28c55d1)


- [ X ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
<!--
List any breaking changes, including namespace, public class/method/field changes, prototype renames; and provide instructions for fixing them. This will be pasted in #codebase-changes.
-->

**Changelog**
<!--
Make players aware of new features and changes that could affect how they play the game by adding a Changelog entry. Please read the Changelog guidelines located at: https://docs.spacestation14.io/en/getting-started/pr-guideline#changelog
-->

<!--
Make sure to take this Changelog template out of the comment block in order for it to show up.
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->

:cl:
- add: Added the Administrative Assistant Role! Assist command in their day-to-day activities, grab the captain a coffee, answer faxes in the bridge!

noctyrnal committed:
file move

noctyrnal committed:
stamp and id

noctyrnal committed:
trying to figure out loadout yml

noctyrnal committed:
fixes loadout and a few borked ftls

noctyrnal committed:
Accidentally deleted courier spawn

oops!

noctyrnal committed:
Adds spawner to every map + updates prototypes

noctyrnal committed:
Merge branch 'master' into admin-assist

Axionyxx reviewed (changes requested):
Why does this remove courier localisation?

You shouldn't be editing maps in the same pr that introduces something, mappers should do that on their own, or if they're unavailable, it should be a seperate pr.

noctyrnal commented:
> Why does this remove courier localisation?

Probably me misreading one of the changes in master and assuming I accidentally deleted something, I'll go undo that now

> You shouldn't be editing maps in the same pr that introduces something, mappers should do that on their own, or if they're unavailable, it should be a seperate pr.

Noted

Axionyxx commented:
I see more problems, but I'll do a bigger review tomorrow

noctyrnal committed:
Reverts Spawns + Fixes mail courier somewhat

noctis makes yet another oopsie, I'll revert / fix whatevers needed after review

DangerRevolution commented on Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/StarterGear/backpack.yml:
```diff
@@ -32,3 +32,16 @@
+  - type: StorageFill
+    contents:
+      - id: BoxSurvival
+      - id: Flash
```
Flash? They're an assistant?

noctyrnal replied:
Was thinking they'd need some self defence on the bridge, but I guess they can just take the funny bridge axe in bad situations. Flash removedm

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Objects/Devices/pda.yml:
```diff
@@ -55,9 +55,9 @@
 
 - type: entity
   parent: BasePDA
-  id: CourierPDA
```
uh?

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Objects/Misc/identification_cards.yml:
```diff
@@ -1,12 +1,14 @@
   parent: IDCardStandard
-  id: CourierIDCard
-  name: courier ID card
+  id: AdminAssistantIDCard
```
uh? (part 2)

DangerRevolution commented on Resources/Prototypes/DeltaV/Roles/Jobs/Command/admin_assistant.yml:
```diff
@@ -0,0 +1,75 @@
+    - !type:DepartmentTimeRequirement
+      department: Command
+      time: 108000 # 30 hours
+    - !type:OverallPlaytimeRequirement 
```
Why is the overall requirement 30 hours, if the Command time *alone* is thirty hours?

noctyrnal replied:
Playtime is now 10 hours overall, 6 hours command.

DangerRevolution commented on Resources/Prototypes/DeltaV/Roles/Jobs/Command/admin_assistant.yml:
```diff
@@ -0,0 +1,75 @@
+    - !type:DepartmentTimeRequirement
+      department: Epistemics 
+      time: 18000 # 5 hours
+    - !type:DepartmentTimeRequirement
```
Common consensus seems to suggest that people think more than 10ish hours for requirement is absurd, might be worth reconsidering this.

DangerRevolution commented on Resources/Prototypes/DeltaV/Roles/Jobs/Command/admin_assistant.yml:
```diff
@@ -0,0 +1,75 @@
+  joinNotifyCrew: true
+  supervisors: job-supervisors-command
+  canBeAntag: false
+  access:
```
This is gross. This is Captain lite, some of these like theatre rooms are *not* relevant to their job.

DangerRevolution commented on Resources/Prototypes/DeltaV/Roles/Jobs/Command/admin_assistant.yml:
```diff
@@ -0,0 +1,75 @@
+  weight: 20
+  startingGear: AdminAssistantGear
+  icon: "JobIconAdminAssitant"
+  requireAdminNotify: true
```
this is not the Captain.

DangerRevolution commented on Resources/Prototypes/DeltaV/Roles/play_time_trackers.yml:
```diff
@@ -5,4 +5,4 @@
   id: JobMedicalBorg
 
 - type: playTimeTracker
-  id: JobCourier
```
erm, what the sigma!?

DangerRevolution commented on Resources/Prototypes/Roles/Jobs/departments.yml:
```diff
@@ -50,6 +50,7 @@
   - HeadOfSecurity
   - ResearchDirector
   - Quartermaster
+  - AdministrativeAssistant # Delta V - AdminAssistant, see Resources/Prototypes/Nyanotrasen/Roles/Jobs/Command/admin_assistant.yml
```
This should also be added to "Station Specific"

DangerRevolution reviewed (commented):
I'm also going to politely beg for you to add localistations for all the new entities <3

Axionyxx commented:
Localized entities are not a requirement and make editing more difficult. It should only be reserved for non English codebases.

DangerRevolution commented:
> Localized entities are not a requirement and make editing more difficult. It should only be reserved for non English codebases.

would be nice for some of our downstreams that aren't English :)

Timemaster99 commented:
By name, this is an assistant role, similar to technical assistant, research assistant, security cadet and medical intern. New players will see this and think of it as a decent role to get them used to the pressures of command before entering a position of responsibility. 
That's why I motion to remove the command time requirement and whitelist. For the whitelist, I believe the purpose of that system is to keep the _most influential_ roles to a higher standard. By itself, this role has no influence and depends on assignments from command to enact any authority (as you will not see anyone respecting the will of someone with "assistant" in their job name). As for the command time, it's unnecessary to require the role with the lowest authority in command to be in the upper echelon of experience for command roles, especially because of how attractive this role is to newer players looking to learn command.
I understand that the requirements are likely a result of the accesses granted to the role. I agree with Danger that these should be limited. Chief Engineer could benefit from AA to repair station-threatening breaches, but they are still not afforded that much power. Command access is all that's required for this role to operate normally, and since that access gives them permission to call the shuttle and make announcements, the 30 hour overall requirement is equitable. Command requirements already average ~30 hours playtime, so this is definitely enough to prevent raiders and uncommitted players.
Aside from that, I think a role to help transition players into command is a great addition. It will help new players attach to a command member of choice and learn from them, while also giving some nice interactions and roleplay for command members themselves. Great job on this PR

noctyrnal committed:
merge master

noctyrnal committed:
Merge branch 'master' into admin-assist

noctyrnal committed:
Review changes

Also fixes the missing courier localisation because I was dumb

noctyrnal commented:
Still need to figure out how to get the map file changes undone fully, but review stuff should be done for now atleast.

noctyrnal committed:
revert engine change

how the fuck did i do it in the first place?

Timemaster99 commented:
> Still need to figure out how to get the map file changes undone fully, but review stuff should be done for now atleast.

`git checkout master FILEPATH` to revert the specified FILEPATH to the version in master. Then commit the changes

Timemaster99 commented on Resources/Locale/en-US/deltav/job/job-description.ftl:
```diff
@@ -1,2 +1,2 @@
 job-description-medical-borg = Half-human, Half-machine. Follow your laws, serve the crew, and assist the medical department.
-job-description-courier = Deliver mail and other packages from and to logistics. Avoid dogs.
```
Courier beware... your description isn't there

noctyrnal replied:
Oops, committing that now.

noctyrnal committed:
Missing courier description

noctyrnal committed:
missing paper_stamp-signature

Timemaster99 commented on Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/StarterGear/duffelbag.yml:
```diff
@@ -19,3 +19,16 @@
+  - type: StorageFill
+    contents:
+      - id: BoxSurvival
+      - id: Flash
```
Duffelbag and Satchel still have a flash, which I believe you intended to remove. Only the backpack has the flash removed

Timemaster99 commented on Resources/Prototypes/DeltaV/Entities/Objects/Devices/pda.yml:
```diff
@@ -81,3 +81,36 @@
+    id: AdminAssistantIDCard
+    state: pda-admin-assistant
+    penSlot:
+      startingItem: PenCap
```
Shouldn't have Captain's pen if they aren't Captain. I'd suggest the default pen or the luxury pen if you're inclined

noctyrnal replied:
Gave them a luxury pen instead

Timemaster99 commented on Resources/Prototypes/DeltaV/Roles/Jobs/Command/admin_assistant.yml:
```diff
@@ -0,0 +1,32 @@
+      time: 21600 # 6 hours
+    - !type:OverallPlaytimeRequirement 
+      time: 36000 # 10 hours
+    - !type:WhitelistRequirement
```
If RP-focused roles like HoP and Lawyer aren't whitelisted, then this shouldn't be either. No particular danger associated with this role if they have command hours

DangerRevolution replied:
I think it's safe to introduce new roles via whitelist and then expand them out to other players but :shrug:

noctyrnal replied:
Took out the whitelist for now

noctyrnal committed:
Review changes II

Remove flashes, change pen, remove whitelist

noctyrnal committed:
Resolves loadout group conflict

noctyrnal committed:
Merge branch 'master' into admin-assist

noctyrnal committed:
removes outdated bureaucracy.rsi content

noctyrnal commented:
I plan on renaming this to Administrative Secretary to further get the point across that this is a support command role and not a command role designed for learning the aspects of command. I'll do this once I have everything all done as per the final review.

noctyrnal committed:
Removes outdated parts of role_loadouts

I accidentally left in 2 empty groups

DangerRevolution commented on Resources/Locale/en-US/deltav/job/job-description.ftl:
```diff
@@ -1,2 +1,3 @@
 job-description-medical-borg = Half-human, Half-machine. Follow your laws, serve the crew, and assist the medical department.
 job-description-courier = Deliver mail and other packages from and to logistics. Avoid dogs.
+job-description-admin-assistant = Assist command in their day-to-day activities, grab the captain a coffee, answer faxes in the bridge.
```
```suggestion
job-description-admin-assistant = Assist command in their day-to-day activities, grab the captain a coffee, answer faxes in the bridge.

```
files should have a line at the end, it's good form supposedly

DangerRevolution commented on Resources/Locale/en-US/deltav/job/job-names.ftl:
```diff
@@ -1,2 +1,3 @@
 job-name-medical-borg = Medical Cyborg
 job-name-courier = Courier
+job-name-admin-assistant = Administrative Assistant
```
```suggestion
job-name-admin-assistant = Administrative Assistant

```
files should have a line at the end, it's good form supposedly

DangerRevolution commented on Resources/Locale/en-US/deltav/job/job-supervisors.ftl:
```diff
@@ -0,0 +1 @@
+job-supervisors-command = all command officials
```
```suggestion
job-supervisors-command = all command officials

```
Two things: Not sure how "all command officials" sounds, maybe "staff" would be better? Not sure.
This should also be checked that this appears okay in a sentence; so just load up the game on a map as an admin assistant and check the flavor text to see it looks okay in the orange starter text you get when you first join as the role

files should have a line at the end, it's good form supposedly

noctyrnal replied:
Changed to all command staff, looks fine in game.

DangerRevolution commented on Resources/Locale/en-US/deltav/paper/stamp-component.ftl:
```diff
@@ -0,0 +1 @@
+stamp-component-stamped-name-admin-assistant = Administrative Assistant
```
```suggestion
stamp-component-stamped-name-admin-assistant = Administrative Assistant

```
files should have a line at the end, it's good form supposedly

DangerRevolution commented on Resources/Locale/en-US/deltav/preferences/loadout-groups.ftl:
```diff
@@ -84,3 +84,9 @@ loadout-group-martial-artist-shoes = Martial Artist shoes
+loadout-group-admin-assistant-head = Administrative Assistant head
+loadout-group-admin-assistant-jumpsuit = Administrative Assistant jumpsuit
+loadout-group-admin-assistant-outerclothing = Administrative Assistant outer clothing
+loadout-group-admin-assistant-back = Administrative Assistant backpack
```
```suggestion
loadout-group-admin-assistant-back = Administrative Assistant backpack

```
files should have a line at the end, it's good form supposedly

DangerRevolution commented on Resources/Locale/en-US/job/job-names.ftl:
```diff
@@ -49,6 +49,7 @@ job-name-visitor = Visitor
 
 # Role timers - Make these alphabetical or I cut you
 JobAtmosphericTechnician = Atmospheric Technician
+JobAdministrativeAssistant = Administrative Assistant
```
This needs to be in a Delta file or have a Delta tag; you can't add tags in .ftl but the game doesn't shit bricks if you add one though.

noctyrnal replied:
Moved to Delta V's job-names.ftl

DangerRevolution commented on Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/StarterGear/backpack.yml:
```diff
@@ -32,3 +32,15 @@
+  - type: StorageFill
+    contents:
+      - id: BoxSurvival
+      - id: RubberStampAdminAssistant
```
```suggestion
      - id: RubberStampAdminAssistant

```
files should have a line at the end, it's good form supposedly

DangerRevolution commented on Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/StarterGear/backpack.yml:
```diff
@@ -32,3 +32,15 @@
+- type: entity
+  noSpawn: true
+  parent: ClothingBackpackCaptain
+  id: ClothingBackpackAdminAssistantFilled
```
Null will tell you not to but I still believe this should be localised; cool RP role that could be used elsewhere :)

noctyrnal replied:
Going to leave them unlocalised for now unless I see more interest in it being done.

DangerRevolution commented on Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/StarterGear/duffelbag.yml:
```diff
@@ -19,3 +19,15 @@
+
+- type: entity
+  noSpawn: true
+  parent: ClothingBackpackDuffelCaptain
```
same comment about localising

DangerRevolution commented on Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/StarterGear/satchel.yml:
```diff
@@ -19,3 +19,15 @@
+- type: entity
+  noSpawn: true
+  parent: ClothingBackpackSatchelCaptain
+  id: ClothingBackpackSatchelAdminAssistantFilled
```
same comment about localising

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Clothing/Ears/headset_alt.yml:
```diff
@@ -0,0 +1,14 @@
+- type: entity
+  parent: ClothingHeadsetAlt
+  id: ClothingHeadsetAltAdminAssistant
+  name: administrative assistant over-ear headset
```
same comment about localising

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Clothing/Ears/headset_alt.yml:
```diff
@@ -0,0 +1,14 @@
+  components:
+  - type: ContainerFill
+    containers:
+      key_slots:
```
worth considering, would they be more useful as assistants if they could access other channels? Or are we expecting heads of staff to hand out keys when needed?

noctyrnal replied:
Going to leave it to departments to hand out keys when needed, since most of their responsibilities can be done with the command and common channels. If they really need the keys after live testing though, Ill PR them getting more keys.

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Clothing/Uniforms/jumpskirts.yml:
```diff
@@ -93,3 +93,14 @@
   - type: Clothing
     sprite: DeltaV/Clothing/Uniforms/Jumpskirt/centcom_officer.rsi
+
+- type: entity
```
same comment about localising

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Clothing/Uniforms/jumpsuits.yml:
```diff
@@ -253,4 +253,15 @@
\ No newline at end of file
+    sprite: DeltaV/Clothing/Uniforms/Jumpsuit/chemshirtsuit.rsi
+
+- type: entity
```
same comment about localising

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Markers/Spawners/jobs.yml:
```diff
@@ -25,3 +26,15 @@
     - sprite: DeltaV/Markers/jobs.rsi
       state: courier
+
+- type: entity
```
this should be on noSpawn: true

DangerRevolution replied:
> Why so?

my brain rotted temporarily and forgot this is a job marker and not an antag marker

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Markers/Spawners/jobs.yml:
```diff
@@ -1,3 +1,4 @@
+
```
what the sigma?

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Objects/Devices/pda.yml:
```diff
@@ -82,3 +82,36 @@
+- type: entity
+  parent: BasePDA
+  id: AdminAssistantPDA
+  name: administrative assistant PDA
```
same comment about localising

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Objects/Misc/identification_cards.yml:
```diff
@@ -10,3 +10,18 @@
+
+- type: entity
+  parent: IDCardStandard
+  id: AdminAssistantIDCard
```
same comment about localising

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Objects/Misc/rubber_stamp.yml:
```diff
@@ -11,3 +11,17 @@
+  name: administrative assistant rubber stamp
+  parent: RubberStampBase
+  id: RubberStampAdminAssistant
+  suffix: DO NOT MAP
```
this should be on noSpawn: true if it's set to "do not map". Same with all the other stamps really I guess.

Axionyxx replied:
No, this should remain spawnable. This is a notice towards mappers to not have this mapped anywhere by default, it should still remain within the spawn menu.

DangerRevolution commented on Resources/Prototypes/DeltaV/Loadouts/Jobs/Command/admin_assistant.yml:
```diff
@@ -0,0 +1,47 @@
+- type: startingGear
+  id: AdminAssistantDuffel
+  equipment:
+    back: ClothingBackpackDuffelAdminAssistantFilled
```
```suggestion
    back: ClothingBackpackDuffelAdminAssistantFilled

```

DangerRevolution reviewed (commented):
looks good; only thing I'd say is to add empty lines at the bottom of each file for consistency; and to localise some of the new entities but those are both up to you.

noctyrnal committed:
Review Changes III

Added empty lines to ends of files, moved some ftl stuff to delta v files, added nospawn to marker and rubber stamp.

Axionyxx commented:
> and to localise some of the new entities

You have already been informed we don't do this several times. If you continue to give our contributors bad advice like this, you may be banned from the repository.

Axionyxx commented on Resources/Prototypes/DeltaV/Entities/Markers/Spawners/jobs.yml:
```diff
@@ -25,3 +25,16 @@
+  id: SpawnPointAdminAssistant
+  parent: SpawnPointJobBase
+  name: administrative assistant
+  noSpawn: true
```
This is a spawnpoint and must be mappable

Axionyxx commented on Resources/Prototypes/DeltaV/Entities/Objects/Misc/rubber_stamp.yml:
```diff
@@ -11,3 +11,18 @@
+  name: administrative assistant rubber stamp
+  parent: RubberStampBase
+  id: RubberStampAdminAssistant
+  noSpawn: true
```
Why is this noSpawn? Having do not map is sufficient. Admins should still be able to spawn it, and this'd make it more difficult to add to the listening post.

DangerRevolution replied:
Listening Post should have an entity with entity filled rather than being mapped in manually

Axionyxx reviewed (changes requested):
I still hate how stamps must be in the same upstream file. I'll look into if this is something we can change sometime

DangerRevolution commented:
> If you continue to give our contributors bad advice like this,

![image](https://github.com/DeltaV-Station/Delta-v/assets/142105406/5c035944-cdf9-477e-9df5-a8b14708a455)

Timemaster99 commented on Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/StarterGear/backpack.yml:
```diff
@@ -32,3 +32,16 @@
+
+- type: entity
+  noSpawn: true
+  parent: ClothingBackpackCaptain
```
This should parent off of the default `ClothingBackpack` for the same reason as the other removed Captain items. Also applies to the duffel and satchel

noctyrnal committed:
Reverts nospawns, changes bags

noctyrnal committed:
Merge branch 'master' into admin-assist

noctyrnal commented:
Should be all ready for a review now, merged to master and fixed the conflicts

Axionyxx reviewed (approved):
The code changes look good, but this is going to go through more discussion first. Do not merge until that is done

noctyrnal commented:
we're so back secretary bros

sword-of-stabbing commented:
This would probably be the command role I would play the most. Head of Paperwork, basically. I love bureaucracy.

Timemaster99 commented:
> This would probably be the command role I would play the most. Head of Paperwork, basically. I love bureaucracy.

HoP in shambles

TadJohnson00 commented on Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/StarterGear/backpack.yml:
```diff
@@ -32,3 +32,16 @@
+  parent: ClothingBackpack
+  id: ClothingBackpackAdminAssistantFilled
+  name: administrative assistant's backpack
+  description: Its a backpack build for carrying paper.
```
*"It's", *"built"

TadJohnson00 commented on Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/StarterGear/duffelbag.yml:
```diff
@@ -19,3 +19,15 @@
+  parent: ClothingBackpackDuffel
+  id: ClothingBackpackDuffelAdminAssistantFilled
+  name: administrative assistant's duffel bag
+  description: Its a duffel bag build for carrying paper.
```
*"It's", *"built"

TadJohnson00 commented on Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/StarterGear/satchel.yml:
```diff
@@ -19,3 +19,15 @@
+  parent: ClothingBackpackSatchel
+  id: ClothingBackpackSatchelAdminAssistantFilled
+  name: administrative assistant's satchel
+  description: Its a satchel build for carrying paper.
```
*"It's", *"built"

TadJohnson00 commented on Resources/Prototypes/DeltaV/Entities/Clothing/Ears/headset_alt.yml:
```diff
@@ -0,0 +1,14 @@
+  - type: Sprite
+    sprite: Clothing/Ears/Headsets/command.rsi
+  - type: Clothing
+    sprite: Clothing/Ears/Headsets/command.rsi
```
new line here would be great x3

TadJohnson00 commented on Resources/Prototypes/DeltaV/Loadouts/loadout_groups.yml:
```diff
@@ -178,6 +178,31 @@
+  name: loadout-group-admin-assistant-outerclothing
+  minLimit: 0
+  loadouts:
+  - CaptainWintercoat
```
pretty please change this out for something else. duravest would work fine for now until your or I get to spriting a dedicated coat

TadJohnson00 commented on Resources/Prototypes/DeltaV/Roles/Jobs/Command/admin_assistant.yml:
```diff
@@ -0,0 +1,31 @@
+  description: job-description-admin-assistant
+  playTimeTracker: JobAdminAssistant
+  requirements:
+    - !type:DepartmentTimeRequirement
```
1984 the command time requirement

TadJohnson00 commented on Resources/Prototypes/DeltaV/Roles/Jobs/Command/admin_assistant.yml:
```diff
@@ -0,0 +1,31 @@
+    - !type:DepartmentTimeRequirement
+      department: Command
+      time: 21600 # 6 hours
+    - !type:OverallPlaytimeRequirement 
```
set this to twenty five hours (90000) pretty please

TadJohnson00 commented on Resources/Prototypes/DeltaV/Roles/Jobs/Command/admin_assistant.yml:
```diff
@@ -0,0 +1,31 @@
+    shoes: ClothingShoesBootsLaceup
+    belt: BoxFolderClipboard
+    id: AdminAssistantPDA
+    ears: ClothingHeadsetAltAdminAssistant
```
newline?

TadJohnson00 commented on Resources/Prototypes/DeltaV/Roles/play_time_trackers.yml:
```diff
@@ -15,3 +15,6 @@
   id: JobCourier
+
+- type: playTimeTracker
+  id: JobAdminAssistant
```
new liiine

TadJohnson00 commented on Resources/Textures/DeltaV/Objects/Devices/pda.rsi/pda-admin-assistant.png:
can we get a new PDA sprite for this pretty please??? >.<

Radezolid commented:
Hey @noctyrnal do you mind if i yoink this and finish it?

noctyrnal commented:
> Hey @noctyrnal do you mind if i yoink this and finish it?

Go wild.

Radezolid commented:
Thanks!
